### ISSUE TITLE
Question: How can we localize all texts ?

### ISSUE THREAD

**Issue by tjaskula** [gregnb/mui-datatables#35]
Is there an easy way to localize texts in any language ?

**Comment by gregnb** [comment]
Hey @tjaskula, there is not any localization support as it stands today. It is a feature that is definitely in the future roadmap

**Comment by victorbueno** [comment]
Could we use https://github.com/stefalda/localized-strings ?

**Comment by gregnb** [comment]
@victorbueno I think to start I will provide something in the options that will allow you to start overriding the default language as opposed to storing different language sets 

```
const options = {
   textLabels: {
      pagination: {
         labelRowsPerPage: "Zeilen pro Seite:",
      },
      toolbar: {
        viewColumns: ""
     }
   }
};

```

**Comment by victorbueno** [comment]
@gregnb that's even better!

**Comment by gregnb** [comment]
I'm going to finish this feature today and here's what I have so far for options:

Pass in a property named `textLabels` via options prop

```
      textLabels: {
        pagination: {
          next: "Next Page",
          previous: "Previous Page",
          rowsPerPage: "Rows per page:",
          displayRows: "of" // 1-10 of 30
        },
        toolbar: {
          search: "Search",
          downloadCsv: "Download CSV",
          print: "Print",
          viewColumns: "View Columns",
          filterTable: "Filter Table"
        },
        filter: {
          title: "FILTERS",
          reset: "reset",          
        },
        viewColumns: {
          title: "Show Columns"
        },
        selectedRows: {
          text: "rows(s) deleted",
          delete: "Delete"
        }
    }
```

**Comment by gregnb** [comment]
This issue is resolved in 2.0.0-beta-6 which will also require upgrade to the latest material-ui
